File: javascript/usuarios.js
var usuarios = [
    { id: 1, nombre: "Camila", edad: 22 },
    { id: 2, nombre: "Esteban", edad: 28 },
    { id: 3, nombre: "Felipe", edad: 25 },
    { id: 4, nombre: "Nilo", edad: 26 }
];


for (var n = 0; n < usuarios.length; n++) {
    console.log("Usuario ID: " + usuarios[n].id + ", Nombre: " + usuarios[n].nombre + ", Edad: " + usuarios[n].edad);
}

File: java/usuarios.java
import java.util.ArrayList;

class Usuario {
    // Atributos de la clase Usuario
    String nombre;
    int edad;

    // Constructor de la clase Usuario
    public Usuario(String nombre, int edad) {
        this.nombre = nombre;
        this.edad = edad;
    }

    // Método para mostrar los detalles del usuario
    public void mostrarDetalles() {
        System.out.println("Nombre: " + nombre + ", Edad: " + edad);
    }
}

public class Main {
    public static void main(String[] args) {
        // Crear una lista de usuarios
        ArrayList<Usuario> usuarios = new ArrayList<>();

        // Crear 5 usuarios y agregarlos a la lista
        usuarios.add(new Usuario("Ana", 25));
        usuarios.add(new Usuario("Carlos", 30));
        usuarios.add(new Usuario("Marta", 22));
        usuarios.add(new Usuario("Luis", 35));
        usuarios.add(new Usuario("Julia", 28));

        // Mostrar los detalles de los usuarios
        System.out.println("Detalles de los usuarios:");
        for (Usuario usuario : usuarios) {
            usuario.mostrarDetalles();
        }
    }
}
